spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/security/AuthenticationAuditListener.java

```diff
@@ -51,6 +52,8 @@ public class AuthenticationAuditListener extends AbstractAuthenticationAuditList
 	 */
 	public static final String AUTHENTICATION_SWITCH = "AUTHENTICATION_SWITCH";
 
+	public static final String LOGOUT_SUCCESS = "LOGOUT_SUCCESS";
```

Comment:
Please add some javadoc here that describes the constant. It should also be marked with `@since 3.4.0`.